tests/stix_translation/test_splunk_json_to_stix.py

```diff
@@ -107,14 +107,14 @@ def test_change_cim_to_stix(self):
         assert(user_obj['user_id'] == "ibm_user")
 
         file_obj = TestTransform.get_first_of_type(objects.values(), 'file')
+
         assert(file_obj is not None), 'file object type not found'
-        assert(file_obj.keys() == {'type','parent_directory_ref','created','modified','hashes','name','size'})
```

Comment:
don't remove the hashes here just make sure we pass in the callback so translation works properly.  If you add the code to make a hash unknown then at least we can check unknown.

Reply:
In the line:

        result_bundle = json_to_stix_translator.convert_to_stix(
            data_source, map_data, [data], transformers.get_all_transformers(), options)

just add your callback so we can map file hashes properly

        result_bundle = json_to_stix_translator.convert_to_stix(
            data_source, map_data, [data], transformers.get_all_transformers(), options, callback=hash_type_lookup)

import the function too:

from stix_shifter.stix_translation.src.modules.splunk.splunk_utils import hash_type_lookup

You can add a unit test that tests all 3 hash types.